stats/dashboard.py

```diff
@@ -243,13 +243,18 @@ def valid_value(value_element):
 
 
 def valid_coords(x):
-    coords = x.split(' ')
+    try:
+        coords = x.split(' ')
+    except AttributeError:
+        return False
     if len(coords) != 2:
         return False
     try:
         x = decimal.Decimal(coords[0])
         y = decimal.Decimal(coords[1])
-        if x == 0 and y ==0:
+        if x == 0 and y == 0:
+            return False
+        elif x < -90 or x > 90 or y < -180 or y > 180:
```

Comment:
Also, on the documentation front: might be worth adding a one-line comment to say what this is doing...